code/modules/awaymissions/capture_the_flag.dm

```diff
@@ -440,7 +437,7 @@
 		. = TRUE
 	if(ishuman(target))
 		var/mob/living/carbon/human/H = target
```

Comment:
Change this to a mob typecheck since it now checks the faction and not the worn suit.